.hlint.yaml

```diff
@@ -8,3 +8,7 @@
 
 # this doesn't work well with Rank2Types
 - ignore: {name: Eta reduce}
+- ignore: {name: Unused LANGUAGE pragma}
```

Comment:
Have you checked to see if the suggestions that `hlint` provides here are reasonable? I'm not necessarily opposed to ignoring certain `hlint` warnings, but if the warnings are reasonable, I'd prefer just applying the suggestions.

Reply:
Fair. I cleaned up the other warnings but left `LANGUAGE` one. It has false positives (`DeriveTraversable` not always implies `DeriveFunctor`, e.g.) so I just left it.